test/mailers/previews/receipt_mailer_preview.rb

```diff
@@ -0,0 +1,6 @@
+# Preview all emails at http://localhost:3000/rails/mailers/receipt_mailer
```

Comment:
all emails in the queue? or ever?

Reply:
It generates a sample receipt email based on the first thesis in the system on your dev environment. If you have not, it'll be blank.